pkg/api/api.go

```diff
@@ -113,6 +113,7 @@ var (
 	errFileStore            = errors.New("could not store file")
 	errInvalidPostageBatch  = errors.New("invalid postage batch id")
 	errBatchUnusable        = errors.New("batch not usable")
+	ErrDevNodeNotSupported  = errors.New("dev node not supported")
```

Comment:
I'd change the error to `errUnsupportedDevNodeOperation  = errors.New("operation not supported in dev mode")`. Also, the error doesn't have to be exported, re-export can be done in the `export_test.go` file if needed in tests.